lib/tasks/scripts/correct_bangladesh_medical_histories.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class CorrectBangladeshMedicalHistories
+  def self.call(*args)
+    new(*args).call
+  end
+
+  attr_reader :dryrun, :verbose
+
+  def initialize(dryrun: false, verbose: true)
+    @dryrun = dryrun
+    @verbose = verbose
+  end
+
+  def call
+    unless Rails.application.config.country[:abbreviation] == "BD"
+      log "Cannot run this cleanup outside of Bangladesh"
+      return
+    end
```

Comment:
This is important for this cleanup script, since it would clobber other environments.